qualtran/bloqs/factoring/ecc/ec_phase_estimate_r.py

```diff
@@ -48,27 +50,54 @@ class ECPhaseEstimateR(Bloq):
     Args:
         n: The bitsize of the elliptic curve points' x and y registers.
         point: The elliptic curve point to phase estimate against.
+        ec_window_size: The number of bits in the ECAdd window.
```

Comment:
to my ear, these two parameters are named inconsistently. ECAdd -> ec; ModMul -> Mul.

Reply:
document what happens if window size is 1